openghg/store/_boundary_conditions.py

```diff
@@ -139,7 +143,10 @@ def read_file(
 
         filepath = next(iter(unseen_hashes.values()))
 
-        with open_dataset(filepath) as bc_data:
+        if chunks is None:
```

Comment:
Just to check, is this a placeholder to use for implementing the default chunking scheme in another PR? Or have I misinterpreted that?